package.json

```diff
@@ -21,6 +21,7 @@
   "license": "ISC",
   "devDependencies": {
     "@solana/web3.js": "^1.73.2",
+    "@types/axios": "^0.14.0",
```

Comment:
Do you need this?

Reply:
not strictly necessary but vs code complains if you don't have it